packages/fxa-payments-server/src/routes/Subscriptions/Cancel/en.ftl

```diff
@@ -11,3 +11,17 @@ sub-item-cancel-msg =
 sub-item-cancel-confirm =
     Cancel my access and my saved information within
     { $name } on { $period }
+
+# $promotion_name (String) - The name of the promotion.
+# LOCALIZATION NOTE (sub-promo-coupon-applied): %S will be replaced at run-time
+# The following IDs determined by the <PriceDetails /> component can be used for this placeholder:
+# price-details-tax-${interval},
+# price-details-no-tax-${interval},
+# price-details-tax,
+# price-details-no-tax
+# Examples:
+# 20% OFF coupon applied: $11.20 + $0.35 tax monthly
+# Holiday Offer 2023 coupon applied: $11.20 monthly
+# Cybersecurity Awareness Month 2023 coupon applied: $11.20 + $0.35 tax
+# Summer Promo VPN coupon applied: $11.20
+sub-promo-coupon-applied = { $promotion_name } coupon applied: <priceDetails>%S</priceDetails>
```

Comment:
Hi @xlisachan - thanks for all your cooperation on this string! I was envisioning a fluent placeholder here, `%S` is a gettext placeholder typically, so is a bit out of place for fluent. I realize my link to our documentation uses a gettext string as en example, so apologies for the confusion!

That being said I consulted some other l10n team members about this string to get their advice. In this case, they mentioned it should be okay if we use what we have here - but just remove the %S text. In other words only `<priceDetails></priceDetails>`. Please see my suggestion below - I've also adjusted the comment slightly to remove the gettext style comment.

```suggestion
# $promotion_name (String) - The name of the promotion.
# The <priceDetails><priceDetails> component acts as a placeholder and could use one of the following IDs:
# price-details-tax-${interval},
# price-details-no-tax-${interval},
# price-details-tax,
# price-details-no-tax
# Examples:
# 20% OFF coupon applied: $11.20 + $0.35 tax monthly
# Holiday Offer 2023 coupon applied: $11.20 monthly
# Cybersecurity Awareness Month 2023 coupon applied: $11.20 + $0.35 tax
# Summer Promo VPN coupon applied: $11.20
sub-promo-coupon-applied = { $promotion_name } coupon applied: <priceDetails></priceDetails>
```